src/plugins/timeConductor/ConductorHistory.vue

```diff
@@ -196,11 +247,24 @@ export default {
             return records;
         },
         formatTime(time) {
+            let format = this.timeSystem.timeFormat;
+            let negativeOffset = false;
```

Comment:
let's use the `is` prefix for booleans.

Reply:
updated! (apparently a while ago and I didn't let you know)